core/types/src/privacy.ts

```diff
@@ -0,0 +1,21 @@
+export type Circuit = {
+  id: string;
+  name: string;
+  description?: string;
+  type?: string;
+  provider: string;
+  curve?: string;
+  constraintSystem?: string;
+  metadata?: object;
+}
+
+export type Commitment = {
+  metadata: object; // arbitrary data
+  location: number;
+  salt: string; // salt such that currentHash = H(data + salt)
+  proof: number[];
+  publicInputs: string[];
+  sender: string; // address of participant who created the commitment
+  signatures: object; // allows mapping from participant address -> signature
+  value: string; // values needed for verification (Shield.verifyAndPush inputs)
```

Comment:
The inputs for `Shield.verifyAndPush()` include:
- proof
- publicInputs
- value/commitment
Can we update the comment here to make it clear `value === commitment`?